packages/gatsby-theme-api-docs/src/components/type/properties/rows/name-type.js

```diff
@@ -28,10 +29,37 @@ const NameType = ({ apiKey, property }) => {
     apiKey,
   });
 
+  const isRegex = (regex) =>
+    regex.charAt(0) === '/' && regex.charAt(regex.length - 1) === '/';
```

Comment:
nitpick on naming: the parameter would be better named "string".  When passing, it's unknown whether it's a regex or not.  Or, in other words:  The function provides the anser to the question "is this regex a regex" (which is pointless) :-)

Reply:
True 💯